packages/pipeline/src/pdk-pipeline.ts

```diff
@@ -105,6 +106,7 @@ export class PDKPipeline extends CodePipeline {
       autoDeleteObjects: true,
       removalPolicy: RemovalPolicy.DESTROY,
       encryption: BucketEncryption.S3_MANAGED,
+      accessControl: BucketAccessControl.LOG_DELIVERY_WRITE,
```

Comment:
Not sure if it's worth using IAM and the bucket policy for the access logs bucket instead? (ref: https://github.com/aws/aws-cdk/issues/22183) I think this also requires a cdk.context.json feature flag too though

Reply:
Fixed :)